aries_vcx/src/indy/primitives/credential_definition.rs

```diff
@@ -69,13 +69,17 @@ enum_number!(PublicEntityStateType
 
 #[derive(Clone, Deserialize, Debug, Serialize, PartialEq, Default)]
 pub struct CredentialDef {
-    pub cred_def_id: String,
+    pub cred_def_id: String, // TODO: Remove!
```

Comment:
Found better way actually, we can do 

```
#[serde(alias = "cred_def_id")]
id: String
```
see 
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=d74203f0810d4951fdd3ed4c647236f2